### PR TITLE
Use pytest_operator_cache plugin instead of conftest.py

### PR DESCRIPTION
**Pull request by carlcsaposs-canonical**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -71,13 +71,13 @@ description = Run integration tests for the database relation
...
     {[testenv]pass_env}
     CI
-    CI_PACKED_CHARMS
 deps =
     juju==2.9.38.1
 ;   Force compatible mysql-connector-python version <8.1.0: https://github.com/juju/python-libjuju/issues/914
     macaroonbakery==1.3.1
     pytest
     pytest-operator
+    git+https://github.com/canonical/data-platform-workflows@pytest-plugin-cache\#subdirectory=python/pytest_plugins/pytest_operator_cache
```
Comment (taurus-forever) [inline tox.ini:80]: Are you planing to publish the plugin on PIP or keep this AS IS?

**Review by taurus-forever — COMMENTED** [review]
LGTM in general, one question inside.

**Review by taurus-forever — APPROVED** [review]
I like simplifications! Tnx!

**Review by paulomach — APPROVED** [review]
nice!

**Pull request merged**